content/agent/faq/upgrade-to-agent-v6.md

```diff
@@ -0,0 +1,409 @@
+---
+title: Upgrade to Datadog Agent v6
+kind: faq
+---
+
```

Comment:
IMO this page would deserve a short intro on what it's for, typically:

* This page describes how to upgrade to v6 of the Datadog Agent when Agent v5 is installed
* If the reader is interested in installing the Agent from scratch, we should redirect them to https://app.datadoghq.com/account/settings#agent.
* the intro should have links to the list of changes between agent 5 and 6 (the sentence `To see all changes between Agent v5 and v6, consult the Datadog Agent dedicated changes documentation.` in `agent/_index.md` should IMO be removed from that page and moved to this intro here).

Reply:
also, there should be a TOC at the top here

Reply:
Finally, it'd be ideal to have a similar structure as the downgrade page, i.e. a per-OS/distribution approach first, and then tabs for CLI and Config mgmt tools (feel free to ignore this comment for now, this could be improved separately)

Reply:
You're right, but I think we should do this once we merge the Content in documentation/ repo and the content in datadog-agent/ repo.

Reply:
The ToC is always displayed on each doc page (in the upper right corner of the content, the little book icon).

I'm going to add an intro with links to the corresponding sections thought :) good idea.

Reply:
oh, I never noticed the book icon! Suggestion, completely out of the scope of the PR: is there a way to make the TOC be shown by default? :)

Intro LGTM, thanks 👍